addon/services/cookies.js

```diff
@@ -50,7 +51,10 @@ export default Service.extend({
     return this._filterCachedFastBootCookies(fastBootCookiesCache);
   }).volatile(),
 
-  read(name) {
+  read(name, options = {}) {
+    options = assign({}, DEFAULTS, options || {});
+    assert('Domain, Expires, Max-Age, and Path options cannot be set when reading cookies', isEmpty(options.domain) && isEmpty(options.expires) && isEmpty(options.maxAge) && isEmpty(options.path));
```

Comment:
Can you add tests for this (see e.g. https://github.com/loganrosen/ember-cookies/blob/9392c6d2504835c1bce10ff1503f63e241672972/tests/unit/services/cookies-test.js#L18)?